utility.py

```diff
@@ -0,0 +1,75 @@
+from concurrent import futures
+from azure.storage.blob import BlockBlobService, PublicAccess
+import os, uuid,sys
+import io
+import grpc
+import time
+import os.path
+import config
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+
+CHUNK_SIZE = 1024 * 1024
+
+def download_chunk(file):
+     with open(file, 'rb')as f:
+         while True:
+             chunk = f.read(CHUNK_SIZE)
+             if len(chunk) == 0:
+                 return
+             yield hwsc_file_transaction_svc_pb2.chunk(buffer=chunk)
+
+def get_file_type(file_Name):
+    IMAGES = "images"
+    AUDIOS = "audios"
+    VIDEOS = "videos"
+    FILES = "files"
+
+    image_exts_dict = {"jpg" : IMAGES, "jpeg" : IMAGES, "png" : IMAGES, "bmp" : IMAGES, "tif" : IMAGES, "gif" : IMAGES, "tiff" : IMAGES}
+    audio_exts_dict = {"wav" : AUDIOS, "wma" : AUDIOS, "ogg" : AUDIOS, "m4a": AUDIOS, "mp3" : AUDIOS}
+    video_exts_dict = {"flv" : VIDEOS, "wmv" : VIDEOS, "mov" : VIDEOS, "avi" : VIDEOS, "mp4" : VIDEOS}
+
+    fileList = file_Name.split('.')
+    extension = fileList[-1]
```

Comment:
Variable naming convention in python says to name variables using underscore_separated_names